plugins/proxies/lib_saltlib.ecllib

```diff
@@ -22,7 +22,7 @@ EXPORT GeohashDecodeResultRecord := RECORD
     REAL latitude_err;
     REAL longitude_err;
 END;
-export SaltLib := SERVICE : plugin('saltlib'),fold
+export SaltLib := SERVICE : plugin('saltlib'),fold,time
```

Comment:
So might the cumulative impact of all the calls... these functions are not trivial...

If it was the case that a significant amount of time was spent in these functions, we'd want to know...